packages/next-css/css-loader-config.js

```diff
@@ -81,16 +81,6 @@ module.exports = (
     )
   }
 
-  // When not using css modules we don't transpile on the server
-  if (isServer && !cssLoader.options.modules) {
```

Comment:
This can't be removed, it'd mean that server files get transpiled for nothing, effectively making things slower for no benefit.

Reply:
I understand. My point is that checking just `modules` is wrong. If the user have postcss loader that need to run (on server too) but he doesn't want css modules, he losts both, which is wrong, isn't it? Any transpiler (saas, less,...) is ignore just because user don't want modules.

Also, setting modules to false may be mean that the user don't want local modules by default but global modules. e.g. he could provide this configuration

```
cssModules: false,
  cssLoaderOptions: {
    localIdentName: '[name]_[local]___[hash:base64:5]',
  },
```

which is valid: it means don't consider modules local by default but when they're esplicitely local use that identName.